doc/source/multivariate.rst

```diff
@@ -223,6 +223,11 @@ In addition to the methods listed in the common interface above, we also provide
 
     Write the squared Mahalanbobis distances from each column of x to the center of d to r.
 
+.. function:: rand(rng, d)
```

Comment:
I was more just thinking that you could update the existing lines in "Sampling" (line ~90) section of this file to something like:

```
.. function:: rand([rng], d)

    Sample a vector from the distribution ``d``.

.. function:: rand([rng], d, n)

    Sample n vectors from the distribution ``d``. This returns a matrix of size ``(dim(d), n)``, where each column is a sample.

.. function:: rand!([rng], d, x)

    Draw samples and output them to a pre-allocated array x. Here, x can be either a vector of length ``dim(d)`` or a matrix with ``dim(d)`` rows.
```